app/views/works/_work_blurb.html.erb

```diff
@@ -1,5 +1,5 @@
 <% # expects "work" %>
-<li id="work_<%= work.id %>" class="<% if is_author_of?(work) %>own <% end %><% if work.unrevealed? %>mystery <% end %>work <%= css_classes_for_creation_blurb(work) %>" role="article">
+<li id="work_<%= work.id %>" class="<% if is_author_of?(work) %>own <% end %><% if work.unrevealed? %>mystery <% end %>work <%= css_classes_for_creation_blurb(work) %>" role="article" data-updated-at="<%= work.updated_at.to_i %>">
```

Comment:
The Jira issue is for making this a comment, not an attribute

Reply:
Yeah but I think an attribute would be easier for crawlers to parse. @zz9pzza what do you think?

Reply:
OK but I'm a bit worried that if we ship this we can't really change it later since it will break any existing scripts